.changeset/six-wolves-admire.md

```diff
@@ -0,0 +1,66 @@
+---
+'@backstage/integration': minor
+---
+
```

Comment:
Bit too much text for a changeset here tbh, best just explain what's changed quick and then you can put a reference to the documentation to explain how it all works.

Reply:
I shortened the changeset 👍.